metricshub-snmp-extension-common/src/test/java/org/sentrysoftware/metricshub/extension/snmp/detection/SnmpGetNextCriterionProcessorTest.java

```diff
@@ -140,4 +141,74 @@ public void testProcess_SnmpRequestException() throws Exception {
 
 		assertFalse(result.isSuccess());
 	}
+
+	@Test
+	void testCheckSNMPGetNextExpectedValue_NullResult() throws Exception {
+		String hostname = "hostname";
+		String oid = "1.3.6.1.2.1.1.1.0";
+		String expectedResult = "expectedResult";
+		String result = null;
+
+		Method method =
```

Comment:
Remove this please, use default package visibility for the methods you want to test.